### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
## Checklist:

- [x] You have followed our [**contributing guidelines**](https://github.com/AppStoreFoundation/asf-contracts/wiki/Contributing-to-AppCoins-Protocol-smart-contracts)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [x] Pull request has tests (at least 80% coverage)
- [x] Code is well-commented, linted and follows project conventions
- [x] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved and auto-closes related issues

### CODE REVIEW

File: test/extendedAdvertisement.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ contract('ExtendedAdvertisement', function(accounts) {
   		TestUtils.setAppCoinsInstance(appcInstance);
```
Comment (appcoinsteam) [inline test/extendedAdvertisement.js]: #### Tests:
```diff 
+ 114 passing
```
#### Test coverage: 
83.97 %

**Review by zepcp — APPROVED** [review]

File: test/extendedAdvertisement.js (hunk 1 above)
Comment (appcoinsteam) [inline test/extendedAdvertisement.js]: #### Tests:
```diff 
+ 114 passing
```
#### Test coverage: 
83.97 %

Comment (appcoinsteam) [inline test/extendedAdvertisement.js]: #### Tests:
```diff 
+ 114 passing
```
#### Test coverage: 
83.97 %

**Review by tcrosado — APPROVED** [review]

**Pull request merged**